### DISCUSSION TITLE
Items guide

### DISCUSSION THREAD

**Discussion by ExplodingHairyPotatoCat** [geckoking7/kgt_idle#7] [Ideas]
Add a guide where it shows what items there are, and what drops them. preferably in like a grid or something. (example being pickaxe, dropped by pickaxe, drop chance of whatever drop chance is, followed by the item stats.) would be useful when looking for a specific item, but you don't wanna go searching through the enemies menu for where it is.

**Comment by ExplodingHairyPotatoCat** [comment]
You could also allow people to disable certain item drops if they're cluttering their inventory (too many shoes when griding apple kills for the apple achievement for example)

**Reply by geckoking7** [reply]
Ok, I made an item filter guide thing but it is unlocked from a drop in zone 9, which may seem late but I am going to try and make >90 zones. It also comes with a hp multiplier based on how many items your have collected and how much of each item you have collected.

**Comment by geckoking7** [comment]
That's exactly what I am coding.
Are you in my house?

**Reply by ExplodingHairyPotatoCat** [reply]
maybe.
